test/onnx/test_pytorch_jit_onnx.py

```diff
@@ -78,6 +79,31 @@ def test_example_ir(self):
         b = torch.randn(2, 3)
         self.run_test(graph_ir, (a, b))
 
+    def test_log_softmax(self):
+        graph_ir = """
+        graph(%x: Tensor):
+          %half_to_float: bool = prim::Constant[value=0]()
+          %dim: int = prim::Constant[value=1]()
+          %y = aten::_log_softmax(%x, %dim, %half_to_float)
+          return (%y)
+        """
+        x = torch.randn(5, 2)
+        self.run_test(graph_ir, (x,))
+
+    @unittest.skipIf(
```

Comment:
You can use `@skipIfNoCuda`, from `pytorch_test_common.py`

Reply:
Sure.